tools/autograd/derivatives.yaml

```diff
@@ -73,7 +73,7 @@
 - name: addcdiv(Tensor self, Tensor tensor1, Tensor tensor2, *, Scalar value)
   self: grad
   tensor1: grad * value / tensor2
-  tensor2: -grad * value * tensor1 / (tensor2 * tensor2)
+  tensor2: grad * (self - result) / tensor2
```

Comment:
Isn't this going to end up with the same problems as `addcmul`?

Reply:
I don't think so. `addcmul` is `self + value * tensor1 * tensor2`. Derivatives would be:
- `self` => `grad`
- `tensor1` => `value * tensor2` (And the other way for `tensor2`). These are the best possible (I think).

Reply:
My comment was related to the gist you posted, where you demonstrated that rewriting the derivative in terms of other operations could yield weird sizes with `addcmul`. I'm wondering if the same thing can happen in here

Reply:
It was with `addcdiv`.